packages/generated-react/package.json

```diff
@@ -23,11 +23,11 @@
     "generate:descriptorset": "buf build --path eliza.proto --path example.proto --output dist/descriptorset.bin"
   },
   "dependencies": {
+    "@bufbuild/connect-query": "workspace:^0.1.0",
     "@bufbuild/protobuf": "^1.0.0",
-    "@bufbuild/protoc-gen-connect-query": "*",
-    "@bufbuild/protoc-gen-connect-web": "^0.6.0",
-    "@bufbuild/protoc-gen-es": "^1.0.0",
-    "@bufbuild/connect-query": "*"
+    "@bufbuild/protoc-gen-connect-query": "workspace:^0.1.0",
```

Comment:
in retrospect, this is better in pretty much every way I can think of and I probably should have done this in the first place.  good on `pnpm` for making the improvement automatically!

Reply:
Oh, that was automatic? Very cool. Yeah workspace is a much better prefix to avoid installing a malicious package.